packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -145,6 +145,31 @@ export class MgtPerson extends MgtTemplatedComponent {
     this.requestStateUpdate();
   }
 
+  /**
+   * Fallback when no user is found
+   * @type {IDynamicPerson}
+   */
+  @property({
+    attribute: 'fallback-details',
+    type: Object
+  })
+  public get fallbackDetails(): IDynamicPerson {
+    return this._fallbackDetails;
+  }
+  public set fallbackDetails(value: IDynamicPerson) {
+    if (value === this._fallbackDetails) {
+      return;
+    }
+
+    this._fallbackDetails = value;
+    if (value && value.displayName) {
+      this._personAvatarBg = this.getColorFromName(value.displayName);
+    } else {
+      this._personAvatarBg = 'gray20';
+    }
+    this.requestStateUpdate();
+  }
```

Comment:
If the `personDetails` is already set, we should avoid this since we don't want to change the background then

```suggestion

    if (this.personDetails) {
      return;
    }

    if (value && value.displayName) {
      this._personAvatarBg = this.getColorFromName(value.displayName);
    } else {
      this._personAvatarBg = 'gray20';
    }
    this.requestStateUpdate();
  }
```